### PR TITLE
[PLAYER-2279] Added function to show controls, when we pressed on scr…

### PR DESCRIPTION
**Pull request by IvanSakharovskii**
…een's center

### CODE REVIEW

**Review by MarkEPAM — APPROVED** [review]

**Review by MarkEPAM — APPROVED** [review]

File: sdk/react/ooyalaSkinCore.js [hunk 1]
```diff
@@ -187,23 +197,13 @@ OoyalaSkinCore.prototype.handleVideoTouchMove = function (event) {
...
-    if (isPastAutoHideTime) {
-      this.handleControlsTouch();
-    } else {
-      Log.verbose("handleVideoTouch - Time Zeroed");
-      this.skin.setState({lastPressedTime: new Date(0)})
-    }
-  }
-
-  if (this.skin.state.vrContent){
+  if (this.skin.state.vrContent && event){
```
Comment (MarkEPAM) [inline sdk/react/ooyalaSkinCore.js]: Add space before a brace

File: sdk/react/ooyalaSkinCore.js [hunk 2]
```diff
@@ -187,23 +197,13 @@ OoyalaSkinCore.prototype.handleVideoTouchMove = function (event) {
...
+};
 
 OoyalaSkinCore.prototype.handleVideoTouchEnd = function(event) {
-  function showControlsPanel() {
-    var isPastAutoHideTime = (new Date).getTime() - this.skin.state.lastPressedTime > AUTOHIDE_DELAY;
-    if (isPastAutoHideTime) {
-      this.handleControlsTouch();
-    } else {
-      Log.verbose("handleVideoTouch - Time Zeroed");
-      this.skin.setState({lastPressedTime: new Date(0)})
```
Comment (AlinaVoronkovaEpam) [inline sdk/react/ooyalaSkinCore.js:199]: Should we keep `Log.verbose`?

**Comment by AlinaVoronkovaEpam** [comment]
This method don't break the Android logic of appearing controls, do it?

**Review by AlinaVoronkovaEpam — APPROVED** [review]

File: sdk/react/ooyalaSkinCore.js (hunk 2 above)
Comment (IvanSakharovskii) [reply sdk/react/ooyalaSkinCore.js:199]: this logic didn't change. I just replaced it to the other function.

**Review by atopilskii — APPROVED** [review]

**Pull request merged**